lms/djangoapps/grades/new/subsection_grade.py

```diff
@@ -226,5 +229,5 @@ def _prefetch_scores(self, course_structure, course):
                 course.id, self.student.id, scorable_locations
             )
             self._submissions_scores = submissions_api.get_scores(
-                unicode(course.id), anonymous_id_for_user(self.student, course.id)
+                unicode(course.id), self.student.id
```

Comment:
Confirmed this was a bad idea: https://build.testeng.edx.org/job/edx-platform-python-unittests-pr/23973/testReport/junit/lms.djangoapps.courseware.tests.test_submitting_problems/TestCourseGrader/test_submissions_api_anonymous_student_id/

Will revert.